cmd/run.go

```diff
@@ -503,29 +505,45 @@ func init() {
 	runCmd.Flags().AddFlagSet(runCmdFlagSet())
 }
 
+func createFilesystems() map[string]afero.Fs {
+	// We want to eliminate disk access at runtime, so we set up a memory mapped cache that's
+	// written every time something is read from the real filesystem. This cache is then used for
+	// successive spawns to read from (they have no access to the real disk).
+	// Also initialize the same for `https` but the caching is handled manually in the loader package
+	return map[string]afero.Fs{
+		"file": fsext.NewCacheOnReadFs(
+			fsext.NewUnprependPathFs(afero.NewOsFs(), afero.FilePathSeparator),
```

Comment:
Can you comment in the code why the "unprepend" FS is used, since it's not immediately obvious?